circle.yml

```diff
@@ -127,3 +126,9 @@ test:
             > $CIRCLE_ARTIFACTS/server_file_kms_rawnode.txt
         & bash wait_for_local_port.bash 8000 40
         && ENABLE_KMS_ENCRYPTION=true npm run ft_node
+
+    # Run S3 with multiple data backends ; run multiple_backend_test
+    - S3BACKEND=mem S3DATA=multiple npm start
```

Comment:
instead of npm start, perhaps we should just start the dmd since we don't actually need the s3 server running for these.